typegate/src/services/rest_service.ts

```diff
@@ -24,6 +31,17 @@ export async function handleRest(
     const name = url.pathname.split("/").slice(
       3,
     ).join("/");
+
+    if (name == "__schema") {
+      logger.info(`rest: ${name} fetch openapi schema`);
+      const res = buildOpenAPISpecFrom(url.origin, engine);
+      headers.set("Content-Type", "application/json");
+      return new Response(res, {
+        status: 200,
+        headers,
+      });
+    }
+
```

Comment:
for GET /, let's return them something similar to [this](https://github.com/metatypedev/metatype/blob/main/typegate/src/services/playground_service.ts) with [that](https://redocly.com/docs/redoc/deployment/html/#step-2---reference-the-redoc-script)